### ISSUE TITLE
Mac - Productivity & Vessel Always 0

### ISSUE THREAD

**Issue by maxwell-gh** [Real-Gecko/KSP-SimpleConstruction#2]
KSP version 1.2.1 for Mac OS
MacOS Sierra v10.12.1

It seems no matter the configuration, the Productivity is always locked at 0. I've tried a full blank install with just SimpleConstruction. Even with purely 5 star engineers, right clicking a vessel always shows Productivity and Vessel Productivity/Factor to be 0.  The UI works properly, but no progress is ever made when building craft.

**Comment by Real-Gecko** [comment]
There's issue in EL I believe, I encounter 0 productivity with KPBS modules. I usually solve it like this: quicksave, quickload and then KPBS based vessel productivity is OK.

**Comment by maxwell-gh** [comment]
Solved!
Turns out the problem is with Module Manager as noted [here](https://github.com/sarbian/ModuleManager/commit/8f7fa2698725fb01b8a10ad9c465f2568c64e989).

Module Manager 2.7.4 had a bug where it would not load KerbalTraits.  Since the engineering trait is a KerbalTrait, that explains why productivity was always 0 - no kerbal had the productivity trait loaded.

I updated to Module Manager 2.7.5, and everything works as expected! Keep up the great work!
